src/pcapi/scripts/suspend_fraudulent_pro_users.py

```diff
@@ -0,0 +1,45 @@
+from pcapi.core.bookings.api import cancel_booking_for_fraud
+from pcapi.core.bookings.exceptions import CannotDeleteOffererWithBookingsException
+from pcapi.core.bookings.repository import find_cancellable_bookings_by_offerer
+from pcapi.core.users.api import suspend_account
+from pcapi.core.users.constants import SuspensionReason
+from pcapi.core.users.models import User
+from pcapi.repository.user_offerer_queries import find_one_or_none_by_user_id
+from pcapi.repository.user_queries import find_pro_users_by_email_provider
+from pcapi.scripts.offerer.delete_cascade_offerer_by_id import delete_cascade_offerer_by_id
+
+
+def suspend_fraudulent_pro_by_email_providers(
+    fraudulent_email_providers: list[str], admin_user: User, dry_run: bool = True
+) -> None:
+    fraudulent_pros = []
+
+    for email_provider in fraudulent_email_providers:
+        fraudulent_pros.extend(find_pro_users_by_email_provider(email_provider=email_provider))
+
+    if not dry_run:
+        for fraudulent_pro in fraudulent_pros:
+            try:
+                user_offerrer = find_one_or_none_by_user_id(user_id=fraudulent_pro.id)
+                delete_cascade_offerer_by_id(offerer_id=user_offerrer.offererId)
```

Comment:
micro-détail : mieux vaut n'encadrer dans un try/except que le minimum.

```suggestion
            user_offerrer = find_one_or_none_by_user_id(user_id=fraudulent_pro.id)
            try:
                delete_cascade_offerer_by_id(offerer_id=user_offerrer.offererId)
```